packages/calcite-components/src/components/chip/chip.tsx

```diff
@@ -295,6 +318,19 @@ export class Chip
     }
   };
 
+  private handleSelectionPropertyChange(selected: boolean): void {
```

Comment:
Can we move the sync logic to the parent?

Reply:
I think tabs does something similar, but you definitely don't want to use it as an example. 😅 Can you create a refactor issue for this and assign it to me? We might have other spots doing something similar.